optimum/exporters/openvino/convert.py

```diff
@@ -190,13 +196,17 @@ def export_tensorflow(
 
     library_name = _infer_library_from_model_or_model_class(model=model)
 
+    model_size = None
+    if return_model_size:
+        model_size = calculate_model_size(ov_model)
+
     _save_model(
         ov_model,
         output.parent / output,
         ov_config=ov_config,
         library_name=library_name,
     )
-    return input_names, output_names, True
+    return (input_names, output_names, True), model_size
```

Comment:
I wonder if the `export()` method is used outside of the Optimum code. If so, this change will break backward compatibility.

Reply:
by default return_model_size is False, I changed to add it into output only if this parameter selected to True, so it should not break backward compatibility now